### PR TITLE
Jenkins

### PR DESCRIPTION
**Pull request by jburel**
Download the zip and build using it 
See https://github.com/ome/omero-py/pull/316

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ pipeline {
             steps {
 
                 // Currently disabled. Eventually, this should copy the blitz python zip
-                copyArtifacts(projectName: 'OMERO-build-build', flatten: true, filter: 'version.properties')
+                copyArtifacts(projectName: 'OMERO-build-build', flatten: true)
```
Comment (joshmoore) [inline Jenkinsfile:21]: Does the comment above still apply? or is this now downloaded?

File: Jenkinsfile [hunk 2]
```diff
@@ -32,6 +32,7 @@ pipeline {
                     test -e version.properties
                     # First download of omero-blitz happens here
                     export VERSION_PROPERTIES=${env.WORKSPACE}/version.properties
+                    export ZIP_FILE=${env.WORKSPACE}/omero-blitz-VERSION-python.zip
```
Comment (joshmoore) [inline Jenkinsfile:35]: Is `$` missing from VERSION here?

**Review by joshmoore — COMMENTED** [review]
Few questions

File: Jenkinsfile (hunk 1 above)
Comment (joshmoore) [reply Jenkinsfile:21]: Thanks for the explanation. Can the comment then be removed?

**Review by sbesson — APPROVED** [review]

**Pull request merged**